js/test/test_parse.js

```diff
@@ -1,20 +1,21 @@
-const assert = require('better-assert')
+const assert = require('better-assert');
 
-const { parse } = require('../lib/index')
+const { parse } = require('../lib/index');
 
 describe('parsing', () => {
     it("doesn't parse nonsense", () => {
-        const c = parse('this is total rubbish').component
-        assert(c.type == null)
-    })
+      const c = parse('this is total rubbish').component;
+      assert(c.type == null);
+    });
     it('returns empty object on empty', () => {
-        assert(parse('').component.type == null)
-    })
+      assert(parse('').component.type == null);
+    });
     it('returns unknown type when encountering package size', () => {
-        assert(parse('0603').component.type === 'unknown')
-    })
-})
+      assert(parse('0603').component.type === 'unknown');
+    });
+});
```

Comment:
If you are having issues with the formatting then we could just apply [prettier](https://prettier.io/) to all the JS code.

Reply:
The index.js file uses 2 spaces indentation and has semicolons at the end. Was updating it to match. What's your preference?

Reply:
I added the prettier code formatter, you can apply a uniform style to everything in js/* by doing `npm r fmt`.